odoo/addons/base/views/res_partner_views.xml

```diff
@@ -80,15 +80,15 @@
                     <field name="avatar_128" invisible="1"/>
                     <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "avatar_128"}'/>
                     <div class="oe_title">
-                        <field name="company_type" options="{'horizontal': true}" widget="radio" groups="base.group_no_one"/>
+                        <field name="company_type" options="{'horizontal': true}" widget="radio" groups="base.group_no_one" invisible="context.get('hide_company_type')"/>
```

Comment:
Would rename to something like ``partner_view_hide_company_type``, to prefix with model + use. Does that sound ok for you ?